docs/development/CodingStyle.md

```diff
@@ -30,17 +30,20 @@ Sometimes, for example, you may want other columns and line breaks so it looks l
 Note2: The Astyle settings have been tested and will produce a nice result. Many files will be changed, mostly to the better but maybe not always, so use with care. 
 
 ## Curly Braces
-
-Functions shall have the opening brace at the beginning of the next line.
+### Functions
+Functions shall have the opening and closing braces at the beginning of the next line, and followed by a line break.
```

Comment:
```suggestion
Functions shall have the closing braces at the beginning of the next line, and followed by a line break.
```

Reply:
I feel like this changes the meaning of the original document, this PR was attempting to clarify the existing document without changing its meaning